### ISSUE TITLE
It says its malware and could harm my device

### ISSUE THREAD

**Issue by Floriiixn** [StefanLobbenmeier/youtube-dl-gui#90]
**Is your question related to a problem? Please describe.**

**Describe the question you have.**
A clear and concise question about youtube-dl-gui.

**Additional context**
Add any other context or screenshots about the question here.

**Comment by Floriiixn** [comment]
When i try to download its says it could harm my pc. I downloaded it anyways, so when i try to open, it gets blocked from  Windows defender.

**Comment by StefanLobbenmeier** [comment]
I think last time the issue was that Microsoft defender did not like yt-dlp, but it seems now it is complaining about the whole application? 

Can you add a screenshot of that Microsoft defender popup?

**Comment by StefanLobbenmeier** [comment]
Maybe people also had that issue in the original repo: 

https://github.com/jely2002/youtube-dl-gui/issues?q=Malware

this one in particular: https://github.com/jely2002/youtube-dl-gui/issues/139

**Comment by Floriiixn** [comment]
![1](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/150825524/3ee59f49-c288-4a2b-9032-53a97e7f1d59)
![2](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/150825524/d3bb698e-d0fb-4d76-9fd6-ff6aff44483b)

**Comment by Floriiixn** [comment]
So its nothing to worry about ?

**Comment by StefanLobbenmeier** [comment]
Yeah nothing to worry about. I can just repeat what @jely2002 said in the linked issue, I also would not put malware in this app.

Of course I am also just a stranger on the internet, but maybe the GitHub stars can reassure you that there are around 800 people trusting me.

Also feel free to download the source code, look around it and compile it yourself. The binaries in the releases tab are built by GitHub actions from the source code available here

**Comment by StefanLobbenmeier** [comment]
Not sure if it will help, but I will try uploading the latest version to microsoft for analysis and maybe that will make the warning go away: https://stackoverflow.com/a/66582477

**Comment by StefanLobbenmeier** [comment]
Submitted, I will update the issue again when they finish validating 
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/36509607/f86deaf5-e6b2-4cb4-acb5-a5f88df4bbea)

**Comment by StefanLobbenmeier** [comment]
Got another update last night:

> Analyst comments:
>  The warning you experienced indicates that submitted file(s) did not have dedicated determination in our system. We can confirm that the files submitted are now determined as clean and attempting to download or run the application should no longer show any warnings.  Due to the complexity of our backend systems, it may take up to 12 hours for the determination to be fully reflected and for the SmartScreen warnings to stop.  For further information related to Microsoft Defender SmartScreen please refer to:  https://learn.microsoft.com/en-us/windows/security/operating-system-security/virus-and-threat-protection/microsoft-defender-smartscreen/ 
> Click here for more information

Could you open it again and confirm the notification is gone now? Then I can close this for now

**Comment by EricGuic** [comment]
Hi, I get the same warning just now (see first screencapture). The button says "Do not execute".
If I click on "Informations complémentaires " (=more info), I get another button allowing me to execute the setup.exe.
 
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/81166671/e40a482d-ead2-4ea8-a047-e7ee64d2abf4)

![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/81166671/fe0a59c1-07fc-42c8-9a07-55d7617acf73)

Then, the setup ask me to install the dependencies (C++, ffmpeg, yt-dlp) and then it works :-)

![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/81166671/c9a2b611-e2f5-40a9-b340-a4059035dce9)

**Comment by StefanLobbenmeier** [comment]
> Also feel free to download the source code, look around it and compile it yourself. The binaries in the releases tab are built by GitHub actions from the source code available here

Just to add on this, in v3 I made sure to be as transparent as possible about the releases done through github actions. The checksum is printed during the build which is available to anyone who is logged it, e.g. here: https://github.com/StefanLobbenmeier/yt-dlp-compose/actions/runs/8636702660/job/23677216546. So this makes sure I cannot change out the binaries in the release without anyone noticing.
